grace/models/classifier.py

```diff
@@ -7,48 +7,36 @@
 
 
 class GCN(torch.nn.Module):
-    """A graph convolutional network for subgraph classification.
-
-    Parameters
-    ----------
-    input_dims : int
-        The dimensions of the input.
-    embedding_dims : int
-        The dimensions of the hidden embeddings.
-    output_dims : int
-        The dimensions of the output. This is typically the number of classes in
-        the classifcation task.
-
-    """
+    """A graph convolutional network for subgraph classification."""
 
     def __init__(
         self,
+        input_channels: int,
+        hidden_channels: int,
         *,
-        input_dims: int = 3,
-        embedding_dims: int = 64,
-        output_dims: int = 2,
-    ) -> None:
+        output_classes: int = 2,
```

Comment:
Is there a reason you removed all of this?

Reply:
It was added in https://github.com/alan-turing-institute/grace/pull/72

Reply:
Oh!! sorry will change back to it now

Reply:
So just to be clear; `input_dims` etc is basically the same as `input_channels` (i.e., an integer) and not a full `Tuple` specification of dims?

Reply:
Yeah - although maybe we should keep channels or features rather than dims?

Reply:
torch and torch geometric both use different terminology

Reply:
Ah ok; I think channels probably works better? Dims to me suggests a full shape, but let me know if you disagree!

Reply:
Or even "dim" as a singular works better imo

Reply:
let's revert to channels, but keep the docstrings (updated obv.)

Reply:
@quantumjot ok! and `output_classes` ?

Reply:
this is the number of features coming from the extractor, right? so definitely needs to be documented properly!